src/py21cmfast/wrapper.py

```diff
@@ -2458,6 +2495,9 @@ def run_lightcone(
     coeval_callback_redshifts=1,
     use_interp_perturb_field=False,
     cleanup=True,
+    hooks=None,
+    always_purge: bool = False,
+    minimize_memory_usage=False,
```

Comment:
Should this flag correspond with my own flag for minimising memory? Or do we want these to be different?

I suspect that they should be the same flag, but wanted to double check.

Reply:
Yeah, good point. I'll put this parameter into `UserParams` so it can be propagated into C for your own purposes.